src/chatgpt.ts

```diff
@@ -102,7 +103,7 @@ export class ChatGPTBot {
   async getGPTMessage(text: string, talkerId: string): Promise<string> {
     const conversation = this.getConversation(talkerId);
     try {
-      return await conversation.sendMessage(text);
+      return await conversation.sendMessage(text,{timeoutMs: 2 * 60 * 1000});
```

Comment:
Is that too long? Maybe we can make it the 30s?

Reply:
Can you provide it as a config?

Reply:
> Can you provide it as a config?

Of course, it can be provided as a configuration file.

What is the granularity of the user control in seconds? Or milliseconds?